CDP4ParameterSheetGenerator/Assemblers/OptionSheet/OptionSheetRowAssembler.cs

```diff
@@ -120,6 +120,29 @@ public void Assemble()
             foreach (var nestedElementExcelRow in nestedElementExcelRows)
             {
                 var deeprows = nestedElementExcelRow.GetContainedRowsDeep();
+
+                foreach (var itm in deeprows)
+                {
+                    var splitlevel = itm.ModelCode.Split('\\')[0].Split('.').Length;
+
+                    if (itm.Categories == null)
+                    {
+                        splitlevel += 1;
+                    }
+
+                    var spaces = new string(' ', 3 * Math.Abs(splitlevel - 1));
+
+                    switch (itm)
```

Comment:
Is this `switch` necessary? The cases seems to be doing the same thing`

Reply:
It is since the item is not boxed and the interface has Name property as read only. It cannot be assigned unless we convert to its final type.